repos/fdbt-admin/src/constants.ts

```diff
@@ -10,7 +10,6 @@ export const STATUS_MAP: { [key: string]: string } = {
 };
 
 export const MAIN_USER_POOL_PREFIX = 'fdbt-user-pool';
-
```

Comment:
can we please remove the extra lines added/removed

Reply:
Why does it need putting back? This is me removing whitespace.